pkg/storage/sqlcommon/sqlcommon.go

```diff
@@ -117,6 +124,12 @@ func NewConfig(opts ...DatastoreOption) *Config {
 		cfg.MaxTypesPerModelField = storage.DefaultMaxTypesPerAuthorizationModel
 	}
 
+	if cfg.MaxConcurrency < 0 {
+		cfg.MaxConcurrency = 1
+	} else if cfg.MaxConcurrency == 0 {
+		cfg.MaxConcurrency = 5
```

Comment:
Please make this a const default `defaultMaxReadAuthorizationModelsConcurrency` or something similarly named.